### PR TITLE
Develop layout_catalog

### PR DESCRIPTION
**Pull request by sergey-lipikhin**
- [DEMO LINK](https://sergey-lipikhin.github.io/layout_catalog/)
- [TEST REPORT LINK](https://sergey-lipikhin.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
```
Comment (MaksymMaliuk) [inline src/styles/blocks/card.scss:69]: You can put this in price

File: src/styles/blocks/nav.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    font-weight: 500;
+    font-size: 12px;
+    color: $color-main-accent;
+    cursor: pointer;
+  }
+
+  &__link:hover,
+  .is-active {
+    color: $color-accent-blue;
+  }
```
Comment (MaksymMaliuk) [inline src/styles/blocks/nav.scss:30]: This can be placed into link

**Review by MaksymMaliuk — APPROVED** [review]
Great job, excepting some moments)

File: src/index.html [hunk 3]
```diff
@@ -1,12 +1,442 @@
...
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <ul class="catalog page__catalog">
+        <li class="catalog__item" data-qa="card">
+          <section class="card">
```
Comment (illiarerun) [inline src/index.html:76]: useless container, just create a mix
```suggestion
          <section class="card catalog__item">
```

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
```
Comment (illiarerun) [inline src/styles/blocks/card.scss:3]: box-sizing should be the same for all elements, move it to `*` selector in normalize

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**